src/bci_build/package.py

```diff
@@ -1564,6 +1564,45 @@ def init_container_kwargs(os_version: OsVersion):
     for os_version in (OsVersion.SP4, OsVersion.TUMBLEWEED)
 ]
 
+with open(os.path.join(os.path.dirname(__file__), "grafana", "run.sh")) as entrypoint:
+    _GRAFANA_ENTRYPOINT = entrypoint.read()
+
+GRAFANA_PACKAGE_NAME = "grafana"
+GRAFANA_CONTAINERS = [
+    ApplicationStackContainer(
+        package_name="grafana-image",
+        os_version=os_version,
+        is_latest=os_version in CAN_BE_LATEST_OS_VERSION,
+        name="grafana",
+        pretty_name="Grafana",
+        package_list=[GRAFANA_PACKAGE_NAME],
+        version="%%grafana_version%%",
+        version_in_uid=False,
+        entrypoint=["/run.sh"],
```

Comment:
Is the entrypoint by intention not in `$PATH`?

Reply:
Not clear intention but rather followed the pattern from [Grafana repository](https://github.com/grafana/grafana/tree/main/packaging/docker).